cockatrice/src/game/game_state.cpp

```diff
@@ -11,9 +11,9 @@ GameState::GameState(AbstractGame *_game,
                      bool _resuming,
                      int _currentPhase,
                      bool _gameClosed)
-    : QObject(_game), game(_game), secondsElapsed(_secondsElapsed), hostId(_hostId), isLocalGame(_isLocalGame),
+    : QObject(parent), gameTimer(nullptr), secondsElapsed(_secondsElapsed), hostId(_hostId), isLocalGame(_isLocalGame),
       clients(_clients), gameStateKnown(_gameStateKnown), resuming(_resuming), currentPhase(_currentPhase),
-      gameClosed(_gameClosed)
+      activePlayer(0), gameClosed(_gameClosed)
```

Comment:
I don't think it makes sense to initialise this to zero, the default for activePlayer in the game state is -1, to mean no player is the active player, this will be updated only when the game starts, the player needs to not crash when the active player is a random number otherwise the server could send defective game states to crash players